src/PhpPact/Standalone/ProviderVerifier/VerifierProcess.php

```diff
@@ -43,13 +27,10 @@ public function setLogger(LoggerInterface $logger): self
     }
 
     /**
-     * Execute the Pact Verifier Service.
-     *
-     * @param array $arguments
-     * @param int   $processTimeout
-     * @param int   $processIdleTimeout
+     * @param array<int, string> $arguments
+     * @throws \Exception
      */
-    public function run(array $arguments, $processTimeout, $processIdleTimeout)
+    public function run(array $arguments, ?int $processTimeout = null, ?int $processIdleTimeout = null): void
```

Comment:
I think `$processTimeout` and `$processIdleTimeout` are not used. But I think we can remove them in another pull request.